src/libopenrave/plugindatabase.cpp

```diff
@@ -124,21 +133,19 @@ DynamicRaveDatabase::~DynamicRaveDatabase()
 void DynamicRaveDatabase::Init()
 {
     const char* pOPENRAVE_PLUGINS = getenv("OPENRAVE_PLUGINS"); // getenv not thread-safe?
-    if (!pOPENRAVE_PLUGINS) {
-        RAVELOG_WARN("Failed to read environment variable OPENRAVE_PLUGINS");
-        return;
```

Comment:
why do we remove this warning?

Reply:
That warning was added by https://github.com/rdiankov/openrave/pull/1137 but it is wrong. If failed to read, it should be just ignored.

Reply:
shall we just drop the `return;` statement and keep the warning in the logs?

Reply:
`utils::TokenizeString(pOPENRAVE_PLUGINS` will cause segmentation fault then

Reply:
```
    if (!!pOPENRAVE_PLUGINS) {
        utils::TokenizeString(pOPENRAVE_PLUGINS, s_delimiter, vplugindirs);
        ...
    } else {
        RAVELOG_WARN("Failed to read environment variable OPENRAVE_PLUGINS");
    }
```
 ?

Reply:
ah. noticed your comment below, thanks!